task10/index.js

```diff
@@ -31,22 +31,25 @@ const getData = () => {
     });    
 };
 
-const objShow = async () => {
-    loading();
 
-    const value = await getData();
-
-    //【Question】 Can I use try-catch-finally in here?
-    // try {
-    //     getData();
-    // } catch (error) {
-    //     div.textContent = "データを読み込めませんでした";
-    //     console.error(error);
-    // }
-    // finally{
-    //     removeLoading();
-    // };
+const sequence = async () => {
```

Comment:
I don't think "sequence" is appropriate.
It's hard to come up with a good name.
For example, "request","fetchData"
or something like that.

Reply:
I'm very bad at naming things, so thank you for pointing that out. I will study it little by little.
I will fix it with fetchData.